hm-module.nix

```diff
@@ -0,0 +1,49 @@
+self:
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}:
+
+with lib;
+
+let
+  cfg = config.programs.krewfile;
+  finalPackage = self.packages.${pkgs.system}.krewfile;
+  krewfileContent = pkgs.writeText "krewfile" (concatStringsSep "\n" cfg.plugins);
+in
+{
+  options.programs.krewfile = {
+    enable = mkEnableOption "krewfile";
+
+    installKrew = mkEnableOption "installKrew";
```

Comment:
I think it's fine to always install krew, isn't it? If you really don't want to install anything you can still use some empty package.

Reply:
Just tried with `null` and I get the error `error: A definition for option `home-manager.users.brumhard.programs.krewfile.krewPackage' is not of type `package'.` So I guess it's fine.